docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -50,13 +48,11 @@ when the app discovery feature generates the `KeptnApp` resource:
 
 - **apiVersion** -- API version being used.
 - **kind** -- Resource type.
-   Must be set to `KeptnApp`
+   Must be set to `KeptnApp`.
 
-- **metadata**
-  - **name** -- Unique name of this application.
-    Names must comply with the
-    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
-    specification.
+- **metadata** -- Information about the resources.
+  - **name** -- This field is used to specify the unique name of the application or resource being defined. The name must follow the guidelines outlined in the [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) specification.
```

Comment:
Why do you say "the application or resource being defined" rather than sticking to the "Unique name of this application."  But good that you added the link to the k8s guidelines -- it was an oversight that it wasn't there.